api_v1/openapi.yml

```
@@ -0,0 +1,531 @@
+  /products/{product}/cycles/latest/:
+    get:
+      summary: Get the latest release cycle data for the given product.
+      parameters:
+        - name: product
+          in: path
+          description: 'The name of the product.'
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: successful operation
+          headers: {}
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProductCycleResponse'
+
+  /categories:
+    get:
+      summary: List all endoflife.date categories.
+      responses:
+        '200':
+          description: successful operation
+          headers: {}
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UriListResponse'
+
+  /categories/{category}:
+    get:
+      summary: >
+        List all the products referenced on endoflife.date for the given category.
+        Only a subset of each product's data is returned by this endpoint.
+      parameters:
+        - name: category
+          in: path
+          description: 'The name of the category.'
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: successful operation
+          headers: {}
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProductListResponse'
+
+  /tags:
+    get:
+      summary: List all endoflife.date tags.
+      responses:
+        '200':
+          description: successful operation
+          headers: {}
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UriListResponse'
+
+  /tags/{tag}:
+    get:
+      summary: >
+        List all the products referenced on endoflife.date for the given tag.
+        Only a subset of each product's data is returned by this endpoint.
+      parameters:
+        - name: tag
+          in: path
+          description: 'The name of the tag.'
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: successful operation
+          headers: {}
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProductListResponse'
+
+# Responses must be at the end of the list, contain a schema_version property and be suffixed with
+# 'Response' to facilitate maintenance and reading.
+components:
+  schemas:
+    Uri:
+      type: object
+      properties:
+        name:
+          type: string
+          description: Name of the URI
+          example: tags
+        uri:
+          type: string
+          format: uri
+          description: URI
+          example: {{ site.url }}/tags/
+
+    Identifier:
+      type: object
+      properties:
+        name:
+          type: type
```

This should be maybe:

```suggestion
          type: string
```

---

Fixed, thanks @jamietanna !